src/content/docs/index.mdx

```diff
@@ -116,43 +127,143 @@ Engine • revision e81fa131e59506d9f6af2a0cee7de749131f1bf0
 No issues detected!
 ```
 
-## Login
+## Integrate Shorebird
 
-You can login using the `shorebird login` command:
+Once you have registered and installed the CLI, you're ready to use code push!
+
+Start by creating a new Flutter app:
 
 ```sh
-shorebird login
+flutter create my_shorebird_app
 ```
 
-Example output:
+As with any Flutter app, you can verify this created the standard Counter app by
+following the instructions printed by `flutter create`:
+
+```sh
+cd my_shorebird_app
+flutter run
+```
+
+### Initialize Shorebird
+
+To make this a Shorebird app, run:
+
+```sh
+shorebird init
+```
+
+This will create a `shorebird.yaml` file in the root of your project. This file
+contains your Shorebird `app_id`.
```

Comment:
```suggestion
This will create a `shorebird.yaml` file in the root of your project. This file
contains your Shorebird `app_id`. Your `app_id` is not secret and can be
checked into source control and freely shared.
```